### ISSUE TITLE
Ошибка при создании платежа

### ISSUE THREAD

**Issue by debian17** [yandex-money/yandex-checkout-android-sdk#7]
Здравствуйте. У меня есть "<Ключ для клиентских приложений>", который я получил в тестовом магазине Яндекс.Кассы. Ключ необходим [согласно документации.](https://kassa.yandex.ru/docs/client-sdks/#sdk-dlq-android)

Код создания платежа:

class PaymentActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_payment)
        Checkout.attach(supportFragmentManager)

        Checkout.setResultCallback(object : Checkout.ResultCallback {
            override fun onResult(paymentToken: String, type: PaymentMethodType) {
                Log.e("TOKEN", paymentToken)
            }
        })

        val paymentMethods = Collections.singleton(PaymentMethodType.BANK_CARD)
        Checkout.tokenize(this, Amount(BigDecimal("10.0"), Currency.getInstance("RUB")),
                ShopParameters(getString(R.string.app_name), getString(R.string.payment_subtitle),
                        BuildConfig.YANDEX_APP_KEY, paymentMethods, false, null, null, true))
    }

    override fun onDestroy() {
        super.onDestroy()
        Checkout.detach()
    } }

BuildConfig.YANDEX_APP_KEY - ключ, выданный в тестовом магазине для проведения платежей в этом магазине.

При попытке выполнить данный код появляется [следующая форма.](https://prnt.sc/khx4mo) Нажатие "Try again" приводит к показу загрузки и снова отображению данной формы.

Подскажите, пожалуйста, я что-то упустил или это ошибка на стороне SDK?

Дполнительная информация:
Язык телефона: английский.
В проекте подключен [YandexAuthSdk.](https://tech.yandex.ru/mobileauthsdk/doc/sdk/concepts/android/2.1.0/sdk-android-install-docpage/) Без его подключения данный код приводит к закрытию приложения(см. [issues#6](https://github.com/yandex-money/yandex-checkout-android-sdk/issues/6)). Авторизация с помощью YandexAuthSdk проходит без каких-либо проблем. Я могу получить объект YandexAuthToken [согласно документации](https://tech.yandex.ru/mobileauthsdk/doc/sdk/concepts/android/2.1.0/sdk-android-use-docpage/). Однако, в [документации к SDK Яндекс.Кассы](https://kassa.yandex.ru/docs/client-sdks/#sdk-dlq-android) я не нашел упоминаний о необходимости самостоятельно получить авторизационный токен для дальнейшего его использования.
Прошу Вашей помощи в этом вопросе.

**Comment by kavorka-cat** [comment]
Я получаю такую же ошибку при аналогичных условиях, но в моем случае магазин и выпускаемые ключи - нетестовые. Будут ли какие-либо комментарии от разработчиков по данной проблеме?

**Comment by lynnfield** [comment]
Здравствуйте.
Посмотрим Ваш кейс на этой неделе, разберемся в чем может быть проблема. 
Пока можете проверить, что у Вас не включён тестовый режим `Checkout.configureTestMode(...)` и в настройках магазина в Яндекс.Кассе разрешён платёж произвольной банковской картой.

**Comment by lynnfield** [comment]
@debian17 @kavorka-cat можете дать applicationId ваших приложений?

**Comment by z0mb1ek** [comment]
@lynnfield столкнулись с такой же проблемой

**Comment by lynnfield** [comment]
@z0mb1ek
Напишите нам в поддержку. Укажите applicationId. Мы посмотрим что пошло не так.
В ближайшей версии уберём обфускацию и добавим логирование, чтобы было проще разбираться с проблемами.

**Comment by givotnoe** [comment]
Проблема была связана с версией TLS на pre-Lolipop девайсах. Один из вариантов лечения - https://developer.android.com/training/articles/security-gms-provider
